INSTALL.md

```diff
@@ -31,6 +31,8 @@ CONFIG_HAVE_BPF_JIT=y
 CONFIG_BPF_EVENTS=y
 ```
 
+For Linux kernel version 4.7 or newer, the flag `CONFIG_HAVE_BPF_JIT` has been replaced by `CONFIG_HAVE_EBPF_JIT` (for x86, arm64 and s390) or by `CONFIG_HAVE_CBPF_JIT` (for arm, mips, powerpc and sparc.)
```

Comment:
If one checkout kernel source code and build it by default configuration for either x86_64 or for arm, the `CONFIG_BPF_JIT` is NOT on by default. I double checked this by generating the `.config`  with `make ARCH=x86_64 defconfig` and `make ARCH=arm versatile_defconfig`. 

You're right that `CONFIG_HAVE_(E|C)BPF_JIT` is enabled by default. However, the existing `INSTALL.md` misleads by guiding people to check config flag of `CONFIG_HAVE_BPF_JIT` cause the flag wasn't exist anymore in newest kernel:

https://github.com/iovisor/bcc/blame/master/INSTALL.md#L29

Another way to avoid the misleading is, I think, to delete `CONFIG_HAVE_BPF_JIT` from INSTALL.md since the flag is not selectable by users. Just don't think it's a good idea to provide an outdated information here.